engine/src/main/java/org/terasology/rendering/dag/stateChanges/BindFbo.java

```diff
@@ -61,22 +56,17 @@ public StateChange getDefaultInstance() {
 
     @Override
     public int hashCode() {
-        return Objects.hashCode(fboName);
+        return Objects.hashCode(fboId);
     }
 
     @Override
     public boolean equals(Object obj) {
-        return (obj instanceof BindFbo) && fboName.equals(((BindFbo) obj).fboName);
-    }
-
-    @Override
-    public void update() {
-        fboId = fboManager.get(fboName).fboId;
+        return (obj instanceof BindFbo) && fboId == ((BindFbo) obj).fboId;
     }
 
     @Override
     public String toString() { // TODO: used for logging purposes at the moment, investigate different methods
-        return String.format("%30s: %s (fboId:%s)", this.getClass().getSimpleName(), fboName, fboId);
+        return String.format("%30s: fboId: %s", this.getClass().getSimpleName(), fboId);
```

Comment:
Hmmmm. I just noticed this change and it isn't good: from a human perspective having the fbo id alone in the logger output is of limited use. Having the name printed out is quite important. 

I can imagine a couple of solution:
1. we pass the fbo's URN on construction, solely for the purpose of printing it out in the log alongside the fboId
2. we store the URN in the FBO, but I'm not sure if this is going to come and bite us in the future.